### PR TITLE
NTP-564: tuition partner organisation type

### PR DESCRIPTION
**Pull request by stephenjmcneill**
## Changes proposed in this pull request

Added organisation type to tuition partner page

Organisation-type

## Link to Jira ticket

https://dfedigital.atlassian.net/browse/NTP-564

## Things to check

- [x] Title is in the format "NTP-XXX: Short description" where NTP-XXX is the Jira ticket reference
- [x] Code and tests follow the [coding standards](/docs/coding-standards.md)
- [x] `dotnet format` has been run in the repository root
- [x] Test coverage of new code is at least 80%
- [ ] All UI related acceptance criteria specified in the ticket have been added to the relevant [cypress test feature file(s)](/UI/cypress/e2e/)
- [ ] Unless explicitly mentioned in the ticket, all UI work should have been tested as working on desktop and mobile resolutions with JavaScript on and off
- [ ] Database migrations can be applied to the current codebase in main without causing exceptions
- [ ] Debug logging has been added after all logic decision points
- [ ] All [automated testing](/README.md#testing) including accessibility and security has been run against your code
- [ ] **PR deployment has been signed off by wider team**

### CODE REVIEW

File: UI/Pages/SearchModel.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿using System.ComponentModel;
 using System.Diagnostics.CodeAnalysis;
 using System.Text.RegularExpressions;
+using Google.Apis.Drive.v3.Data;
```
Comment (benagain) [inline UI/Pages/SearchModel.cs:4]: I don't think we should couple the search model to a Google API class.

Comment (stephenjmcneill) [reply UI/Pages/SearchModel.cs]: Removed

**Review by benagain — APPROVED** [review]

**Review by iterative-it — CHANGES_REQUESTED** [review]

File: UI/cypress/e2e/tuition-partner-details.js [hunk 2]
```diff
@@ -172,3 +172,12 @@ Then("the tuition cost information states declares differences", () => {
   cy.get('[data-testid="pricing-differences-for-subjects').should("exist");
   cy.get('[data-testid="pricing-same-for-subjects').should("not.exist");
 });
+
+Then("all tuition partner details are populated correctly", () => {
+  cy.get('[data-testid="results-subjects"] > li:first')
```
[the remark this answers is not in the document]
Comment (stephenjmcneill) [reply UI/cypress/e2e/tuition-partner-details.js:177]: Changed as advised

File: UI/Pages/SearchModel.cs [hunk 3]
```diff
@@ -12,10 +12,13 @@ public SearchModel(SearchModel model) : base()
         Subjects = model.Subjects;
         TuitionType = model.TuitionType;
         KeyStages = model.KeyStages;
+        LegalStatus = model.LegalStatus;
     }
 
     public string? Postcode { get; set; }
 
+    public string? LegalStatus { get; set; }
```
[the remark this answers is not in the document]
Comment (stephenjmcneill) [reply UI/Pages/SearchModel.cs]: Removed

**Review by iterative-it — APPROVED** [review]

**Pull request merged**